tests/test-motion.sh

```diff
@@ -77,6 +77,9 @@ test_detect_motion_reports_motion() {
 	for motion_result in detect_motion():
 	    assert bool(motion_result) == motion_result.motion
 	    if motion_result:
+	        # videotestsrc has motion in bottom right corner:
+	        assert motion_result.region == stbt.Region(
+	            240, 180, right=320, bottom=240)
```

Comment:
I ran this test 1,000 times and it always passed, so it seems that the random noise generated by videotestsrc will always have some motion along the edge rows/columns of this region. Or maybe videotestsrc is deterministic. All good, in any case.